### PR TITLE
steal.lic: Run performance if you don't hide to steal.

### PR DESCRIPTION
**Pull request by Sarvatt**

### CODE REVIEW

**Comment by simiydr** [comment]
As a general question (and applied to this case) would it be better to start using the `play_song` method introduced in common in PR #3546?

**Comment by Tarhish** [comment]
> As a general question (and applied to this case) would it be better to start using the `play_song` method introduced in common in PR #3546?

Nooo, regretfully it's not done yet.  It works in ;performance if I replace methods with calls to common, but I need to make a couple changes to make sure it works right.  It needs a noclean itself, or possibly a way to decouple playing from cleaning neatly and without wasted actions.

File: steal.lic [hunk 1]
```diff
@@ -52,7 +52,7 @@ class Steal
 
     settings = get_settings
     setup(settings)
-
+    start_script('performance', ['noclean']) unless settings.hide_to_steal
```
Comment (simiydr) [inline steal.lic]: I think we probably want to move starting performance down to below the checks that might cause the script to exit quickly. So, perhaps after line 73?

**Comment by simiydr** [comment]
@Tarhish Understood, thanks!

File: steal.lic (hunk 1 above)
Comment (Tarhish) [reply steal.lic]: Yeah, good catch.  There's also buffs in there, which I think means spellcasters with stealing buffs will start playing, try to buff, stop playing, buff, then begin stealing.  73 looks like good to me?

@Sarvatt got any thoughts?

Comment (Sarvatt) [reply steal.lic]: bput handles stopping playing and the script will start again between buffs so it wouldn't hurt anything, but I moved it anyway just to be extra safe :D

**Comment by simiydr** [comment]
Thanks @Sarvatt! Looks good to me.

**Pull request merged**